Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_-_First_SSO_Access.yml

```diff
@@ -0,0 +1,1906 @@
+      name: ""
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "2"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 267.5,
+          "y": -1850
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "1":
+    id: "1"
+    taskid: e337418e-95d8-4cac-8585-f9f2f19daf46
+    type: regular
+    task:
+      id: e337418e-95d8-4cac-8585-f9f2f19daf46
+      version: -1
+      name: IP Enrichment
+      description: IP to check.
+      script: '|||ip'
+      type: regular
+      iscommand: true
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "111"
+    scriptarguments:
+      ip:
+        complex:
+          root: inputs.IPAddress
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 480,
+          "y": -1570
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    fieldMapping:
+    - incidentfield: Source IP
+      output:
+        complex:
+          root: IP
+          accessor: Address
+    - incidentfield: ASN
+      output:
+        complex:
+          root: IP
+          accessor: ASN
+    - incidentfield: ASN Name
+      output:
+        complex:
+          root: IP
+          accessor: ASOwner
+    - incidentfield: Country Name
+      output:
+        complex:
+          root: IP.Geo
+          accessor: Country
+          transformers:
+          - operator: ConvertCountryCodeCountryName
+            args:
+              country_code:
+                value:
+                  simple: IP.Geo.Country
+                iscontext: true
+              country_name: {}
+    - incidentfield: IP Reputation
+      output:
+        complex:
+          root: DBotScore.[0]
+          accessor: Score
+    - incidentfield: Alert Name
+      output:
+        complex:
+          root: incident.xdrsimilarincidents
+          accessor: name
+          transformers:
+          - operator: FirstArrayElement
+    - incidentfield: Country Code
+      output:
+        complex:
+          root: IP.Geo
+          accessor: Country
+    skipunavailable: false
+    quietmode: 2
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "2":
+    id: "2"
+    taskid: a47c2c3e-ddf0-49d2-8e6c-a8b5b11212ea
+    type: title
+    task:
+      id: a47c2c3e-ddf0-49d2-8e6c-a8b5b11212ea
+      version: -1
+      name: Analysis
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "1"
+      - "93"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 267.5,
+          "y": -1715
+        }
+      }
+    note: false
+    timertriggers:
+    - fieldname: containmentsla
```

Comment:
Why the containmentsla starts here? (analysis)

Reply:
Done.